users.js

```diff
@@ -112,10 +193,28 @@ function ipSearch(ip, table) {
 	return false;
 }
 function checkBanned(ip) {
-	return ipSearch(ip, bannedIps);
+	var checkResult = ipSearch(ip, bannedIps);
+	if (!checkResult) return false;
+	if (Array.isArray(checkResult)) {
```

Comment:
The result can be <code>'#cflood'</code> if the IP is banned by the ResourceMonitor and <code>'#ipban'</code> if the IP is banned with /banip or in the ipbans file